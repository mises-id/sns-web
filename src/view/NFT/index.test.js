/*
 * @Author: lmk
 * @Date: 2021-07-15 12:51:10
 * @LastEditTime: 2021-07-15 12:51:20
 * @LastEditors: lmk
 * @Description: UserInfotest
 */
import React from 'react';
import NFTPage from './';
import {shallow } from 'enzyme';
describe('NFTPage', () => {
  it('should render correctly', () => {
    const component = shallow(<NFTPage />)
    expect(component).toMatchSnapshot();
  });
});