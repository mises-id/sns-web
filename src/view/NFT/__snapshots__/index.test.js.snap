// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NFTPage should render correctly 1`] = `
<div>
  <Navbar
    title="MyNFTPageTitle"
  />
  <ForwardRef(PullList)
    data={Array []}
    emptyTxt="NFTEmpty"
    load={[Function]}
    renderView={[Function]}
  />
</div>
`;
